From c7d90bb19a4b22dfb3ff13a28051edef70f1dd19 Mon Sep 17 00:00:00 2001
From: Raghu Udiyar <raghusiddarth@gmail.com>
Date: Fri, 5 Feb 2016 22:30:11 +0530
Subject: [PATCH 37/67] BUG/MINOR: stats: fix missing comma in stats on agent
 drain

The csv stats format breaks when agent changes server state to drain.
Tools like hatop, metric or check agents will fail due to this. This
should be backported to 1.6.
(cherry picked from commit 0d6b7a490e1ed32424dc7c9b0d0aa3cdb8514a8e)
---
 src/dumpstats.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index 3cd54f1..a347a71 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -3546,7 +3546,7 @@ static int stats_dump_sv_stats(struct stream_interface *si, struct proxy *px, in
 			[SRV_STATS_STATE_NOLB]			= "NOLB,",
 			[SRV_STATS_STATE_DRAIN_GOING_DOWN]	= "DRAIN %d/%d,",
 			[SRV_STATS_STATE_DRAIN]			= "DRAIN,",
-			[SRV_STATS_STATE_DRAIN_AGENT]		= "DRAIN (agent)",
+			[SRV_STATS_STATE_DRAIN_AGENT]		= "DRAIN (agent),",
 			[SRV_STATS_STATE_NO_CHECK]		= "no check,"
 		};
 
-- 
2.4.10

